src/audio/lib/load.c

```diff
@@ -1126,6 +1126,8 @@ void AudioLoad_InitSoundFont(s32 fontId) {
     font->numSfx = entry->shortData3;
 }
 
+extern u8 gSequenceTable[];
```

Comment:
This should go in z64audio.h with the others, and should be retyped to `AudioTable`. I must have missed this in the audio PRs 😔
`gAudioCtx.sequenceTable = (AudioTable*)gSequenceTable;` should happily match with
`gAudioCtx.sequenceTable = &gSequenceTable;` like the others

Reply:
are these standalone declarations still needed or can they come from the header
![image](https://github.com/user-attachments/assets/84e5b9cd-558e-44d6-8f79-5071793e4023)

Reply:
Those can also be removed now

Reply:
done. also I notice that some of these are arrays and some are not, is this intentional
![image](https://github.com/user-attachments/assets/6f7cbc67-4cdc-483a-b66f-59275eeed031)

Reply:
`gSequenceTable` shouldn't be an array, but `gSequenceFontTable` should be

Reply:
thanks. all should be fixed, but im sure ill need to make bss changes